app/styles/_component-variables.scss

```diff
@@ -262,6 +262,11 @@ body {
   // variables for ak-skeleton
   --ak-skeleton-linear-gradient-color-1: var(--neutral-grey-100);
   --ak-skeleton-linear-gradient-color-2: var(--neutral-grey-200);
+
+  // variables for tri-state-checkbox
+  --tri-state-checkbox-overridden-bg: var(--success-light);
+  --tri-state-checkbox-border-color: var(--success-main);
+  --tri-state-reset-icon-bg: var(--common-white);
```

Comment:
this is not a standard component please move it to below body

Reply:
Done